sdk/servicebus/azure-servicebus/README.md

```diff
@@ -399,6 +399,21 @@ logger = logging.getLogger('azure.servicebus')
 logger.setLevel(logging.DEBUG)
 logger.addHandler(handler)
 
+# Advanced usage: Log AMQP connection/session/link names
+class CustomFormatter(logging.Formatter):
+
+    def format(self, record: logging.LogRecord) -> str:
+        if not "amqpConnection" in record.__dict__:
+            record.__dict__["amqpConnection"] = None
```

Comment:
if the values are not supposed to be None, why are they being set so here?

Reply:
they aren't supposed to be None for Otel, but if we're just printing them out/logging to other places, this should be fine I believe. However, you're right in that I'm not sure how exactly this will affect otel, since the same extra params won't be present everywhere.